mods/ra/maps/soviet-07/soviet07.lua

```diff
@@ -239,7 +239,9 @@ IntroSequence = function()
 		timerStarted = true
 	end)
 	Trigger.OnAllKilled(StartingUnits, function()
```

Comment:
Couldn't we just drop this entirely?
As you have a `player.HasNoRequiredUnits()` check in `Tick`.